tests/contrib/openstack/test_os_contexts.py

```diff
@@ -3586,13 +3588,16 @@ def test_wsgi_worker_config_context(self,
             "admin_processes": 2,
             "public_processes": 6,
             "threads": 1,
+            "wsgi_rotation": True,
         }
         self.assertEqual(expect, ctxt())
 
     @patch.object(context, '_calculate_workers')
     def test_wsgi_worker_config_context_user_and_group(self,
                                                        _calculate_workers):
-        self.config.return_value = 1
+        self.config.side_effect = fake_config({
+            'worker-multiplier': 1, 'wsgi-rotation': True
+        })
```

Comment:
We could do with the opposite test here; the default is `True`/`On`, so the handy test would be to set it to `False` to get an `Off` in the config.

Reply:
has changed to False, thanks a lot